scripts/download_full_text.py

```diff
@@ -0,0 +1,67 @@
+import argparse
+import requests
+
+import lxml.etree as ET
+import pandas as pd
+import tqdm
+
+
+def read_id_chunk(id_file, batch_size):
+	""" 
+	Chunk the pandas dataframe so not everything is read into memory
+
+    Keyword Arguments:
+    id_file -- the file to read through pandas
+    batch_size -- the size of each chunk
+    """
+	for id_batch in pd.read_csv(id_file, sep="\t", chunksize=batch_size):
+		yield id_batch
```

Comment:
looks like there could be mixed indentation between spaces and tabs. This often causes problems